src/ResultPage.js

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components/macro'
+import Button from './components/Button'
+import { useState, useEffect } from 'react'
+
+function ResultPage({countyData, resetSearch}) {
+
+    const [countyRank, setCountyRank] = useState('')
+
+    useEffect(() => rankCountyIncidence(), [countyData])
+
+    function rankCountyIncidence() {
+        if(countyData.incidence > 35) {
+            setCountyRank('county-rank-red')
+        } else {
+                setCountyRank('county-rank-green')}
+    }
+
+    return (
+        <ResultPageStyled>
+            <section className={countyRank + " result-wrapper"}>
+                {countyData.incidence > 35 ? 
+                <h2> {countyData.countyName} ist ein Covid-19 Hotspot.</h2> :
+                <h2> {countyData.countyName} ist kein Covid-19 Hotspot.</h2> 
+                }
+                <h3>Die 7-Tage-Inzidenz <br/> liegt bei {countyData.incidence}. </h3>
+                <span>Daten vom {countyData.last_update}</span>
+            </section>
+            <section className="information-wrapper">
+                <a href="https://www.bundesregierung.de/breg-de/themen/coronavirus/corona-bundeslaender-1745198">
+                    <Button text="Regeln der Bundesländer"/>
+                </a>
+                <a href="https://www.bundesregierung.de/breg-de/themen/coronavirus/corona-massnahmen-1734724">
+                    <Button text="FAQ Bundesregierung.de"/>
+                </a>
+                <Button text="Neue Suche" onClick={resetSearch}/>
+            </section>
+        </ResultPageStyled>
+    )
+}
+
+const ResultPageStyled = styled.div`
+
+h2 {
+    margin-bottom: 46px;
+    font-size: 32px;
+}
+
+h3 {
+    margin-bottom: 41px;
+    line-height: 1.5;
+    font-size: 26px;
+}
+
+span {
+    font-weight: 300;
+}
+
+a {
```

Comment:
links should be always recognizable as links. therefore some purists state that the text-decoration shouldn't be ommited widthoud substitute of some kind.